### PR TITLE
Add install instructions for Trisquel

### PR DESCRIPTION
**Pull request by ncorder**
These are the instructions for [Trisquel](https://www.wikipedia.org/wiki/Trisquel) users (Ubuntu derivative).

### CODE REVIEW

File: data/distro.yml [hunk 1]
```diff
@@ -510,6 +510,40 @@
       <p>Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
     </ol>
 
+- name: "Trisuqel"
+  logo: "trisquel.svg"
+  info: >
+    <ol class="distrotut">
+      <li>
+        <h2>Install Flatpak</h2>
+        <p>To install Flatpak on Trisquel simply open the Terminal ( MATE Terminal ) and enter :</p>
```
Comment (AsciiWolf) [inline data/distro.yml]:
```suggestion
        <p>To install Flatpak on Trisquel simply open the Terminal (MATE Terminal) and enter:</p>
```

File: data/distro.yml [hunk 2]
```diff
@@ -510,6 +510,40 @@
...
+        <pre><code>
+          <span class="unselectable">$</span> sudo apt install flatpak
+        </code></pre>
+        <p>Then just press enter. APT may also ask you to confirm you want to install the application.</p>
+      </li>
+      <li>
+        <h2>Install the Software Flatpak plugin</h2>
+        <p>Flatpak does not currently have a plugin for MATE's Add/Remove Software Application but you can install Flatpak apps with a Graphic Interface using GNOME Software:</p>
```
Comment (AsciiWolf) [inline data/distro.yml:527]: Is it recommended (officially supported) to install GNOME Software in Trisquel?

File: data/distro.yml [hunk 3]
```diff
@@ -510,6 +510,40 @@
       <p>Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
     </ol>
 
+- name: "Trisuqel"
```
Comment (AsciiWolf) [inline data/distro.yml]: Typo in the distro name.

**Review by AsciiWolf — CHANGES_REQUESTED** [review]

File: data/distro.yml (hunk 3 above)
Comment (ncorder) [reply data/distro.yml]: Thank you, I will fix that

File: data/distro.yml (hunk 2 above)
Comment (ncorder) [reply data/distro.yml:527]: > Is it recommended (officially supported) to install GNOME Software in Trisquel?

As I understand it Trisquel officially comes preinstalled with a modified of GNOME Install/Synaptic. The package name is `trisquel-app-install`. 

GNOME Software, is not currently supported by Trisquel based on a dispute over its dependence on Snapd. [[1](https://trisquel.info/en/issues/23466)]. 

KDE Discover, is a program I have yet to fully test, but there are issue trackers and forum posts for it as well. [[2](https://trisquel.info/en/issues/28357)] [[3](https://trisquel.info/en/forum/should-discover-be-bundles-triskel)]

_______________________________________________________

Despite not being officially supported, having a GUI tool to install apps can be really good for people who are a bit newer to the GNU/Linux ecosystem. However, I do understand if you wish to only list how to add plugins for officially supported package managers as well. :+1:

Comment (ncorder) [reply data/distro.yml:527]: Triskel is a version of Trisquel with KDE but it doesn't appear to have official support for Discover. [[4](https://trisquel.info/en/forum/triskel-9-updater)]

Let me know what your thoughts are and any way I might be able to better revise my PR.

Comment (AsciiWolf) [reply data/distro.yml:527]: Any chance this PR could be updated to reflect that changes? Thanks!

File: data/distro.yml [hunk 4]
```diff
@@ -510,6 +510,40 @@
...
+        <h2>Add the Flathub repository</h2>
+        <p>To enable <a href="https://flathub.org/">Flathub</a>, run:</p>
+        <pre><code>
+          <span class="unselectable">$</span> flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
+        </code></pre>
+        <p>Or you can also enable flathub by downloading the <a class="btn btn-default" href="https://flathub.org/repo/flathub.flatpakrepo">Flathub repository file</a></p>
+      </li>
+      <li>
+        <h2>Restart</h2>
+        <p>To complete setup, restart your system. Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
```
Comment (AsciiWolf) [inline data/distro.yml:543]: Please add this:
`        <p>Note: graphical installation of Flatpak apps may not be possible with Trisquel.</p>'`

Also, please rewrite the whole Trisquel entry using a new format. For example see how do other entries look like now: https://github.com/flatpak/flatpak.github.io/blob/source/data/distro.yml#L1

**Review by AsciiWolf — CHANGES_REQUESTED** [review]

File: data/distro.yml (hunk 4 above)
Comment (AsciiWolf) [reply data/distro.yml:543]: Oops, sorry, I did misread the previous discussion.

File: data/distro.yml (hunk 2 above)
Comment (AsciiWolf) [reply data/distro.yml:527]: Ping.

File: data/distro.yml [hunk 5]
```diff
@@ -510,6 +510,40 @@
...
+          <span class="unselectable">$</span> sudo apt install gnome-software-plugin-flatpak
+        </code></pre>
+      </li>
+      <li>
+        <h2>Add the Flathub repository</h2>
+        <p>To enable <a href="https://flathub.org/">Flathub</a>, run:</p>
+        <pre><code>
+          <span class="unselectable">$</span> flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
+        </code></pre>
+        <p>Or you can also enable flathub by downloading the <a class="btn btn-default" href="https://flathub.org/repo/flathub.flatpakrepo">Flathub repository file</a></p>
```
Comment (AsciiWolf) [inline data/distro.yml:539]:
```suggestion
        <p>Or you can also enable Flathub by downloading the <a class="btn btn-default" href="https://flathub.org/repo/flathub.flatpakrepo">Flathub repository file</a>.</p>
```

File: data/distro.yml (hunk 2 above)
Comment (AsciiWolf) [reply data/distro.yml:527]: (You will also need to rewrite this PR using a new metadata format that the yaml file uses. See how other entries do look like now: https://github.com/flatpak/flatpak.github.io/blob/source/data/distro.yml#L1)

**Comment by AsciiWolf** [comment]
Closing this PR. It has been inactive for a long time without any of the issues addressed and also the distro.yml format has changed since then. Feel free to submit a new PR with these things resolved.

**Pull request closed without merging**